### PR TITLE
282 create a component for settings page

### LINKED ISSUE #282: create a component for settings page
According to [Figma Mockup] (https://www.figma.com/file/RevVhXP9iyGxmch31H0Tce/Project-Leek?node-id=576%3A0)

- show list of connected speakers
- allow to select a speaker (add speaker to nfc-reader)
- logout current user

### PR DESCRIPTION
**Pull request by schnickalodeon**
closes #282 

- Page ist availlable at localhost:3000/settings (if you are loged in)
  - I will add a Settings button in Home.vue if the branch #292 is merged into master

**The functionality of saving the Speaker will be implemented by @anbraten. in #283 **
Since the save button is not implemented yet.

### CODE REVIEW

File: packages/app/src/views/Settings.vue [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    <footer class="flex text-gray-800 py-5">
+      <span class="flex w-full text-xl">
+        <Button round icon="fas fa-caret-left" class="ml-4 my-auto h-12 w-12" />
+        <Button text="Speichern" center-text class="mx-2 px-3 py-2 text-2xl flex-grow" />
+        <Button
+          text="Abmelden"
+          center-text
+          class="mx-2 px-3 text-2xl flex-grow"
+          @click="logoutUser()"
+        />
```
Comment (JanEberlein) [inline packages/app/src/views/Settings.vue:32]: why is this in the footer? in my mind it would be way more intuitive if this was under the user name

**Review by tcoro — APPROVED** [review]
great work!

I disagree with @JanEberlein , placing it next to the name or below it wouldnt look that great in my oppinion.
But this design is also not flawless, but I cant put my finger on how it could be made better.

Maybe we (or some designer?) might come up with a better Idea later

File: packages/app/src/views/Settings.vue (hunk 1 above)
Comment (schnickalodeon) [reply packages/app/src/views/Settings.vue:32]: I will change that. To be honest i have struggled a bit with three diffrent versions of the figma layout ^^

**Review by julianhahn — APPROVED** [review]
looks nice, logout works.

**Pull request merged**